velox/exec/Task.h

```diff
@@ -681,6 +677,20 @@ class Task : public std::enable_shared_from_this<Task> {
   }
 
  private:
+  // Hook of system-wide running task list.
```

Comment:
do we need to expose these things on the header? Could they be statically defined straight into the .cpp file?

Reply:
This is in private section and linked list hook needs to be part of the host object.